client/src/app/videos/+video-watch/video-watch.component.html

```diff
@@ -19,6 +19,11 @@
           <div class="video-info-channel">
             {{ video.channel.displayName }}
             <!-- Here will be the subscribe button -->
+            <span *ngIf="syndication.isSupported" class="icon icon-syndication" 
```

Comment:
This is the account syndication? So it should be after the account name, not the video channel name

Reply:
At that time I didn't make the difference between account, user and channel… ^^'